h2integrate/converters/nitrogen/simple_ASU_performance.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+import openmdao.api as om
+from attrs import field, define
+
+from h2integrate.core.utilities import BaseConfig, merge_shared_inputs
+from h2integrate.core.validators import range_val
+
+
+@define
+class SimpleASUPerformanceConfig(BaseConfig):
+    """Configuration for ASU model. To represent a cryogenic ASU, it is
+    recommended to set the parameter `efficiency_kWh_pr_kg_N2` to 0.119.
+    To represent a pressure swing absorption ASU, it is
+    recommended to set the parameter `efficiency_kWh_pr_kg_N2` to 0.29.
+    """
+
+    size_from_N2_demand: bool = field()
+    rated_N2_kg_pr_hr: float | None = field(default=None)
+    ASU_rated_power_kW: float | None = field(default=None)
+
+    N2_fraction_in_air: float = field(default=78.11)
+    O2_fraction_in_air: float = field(default=20.96)
+    Ar_fraction_in_air: float = field(default=0.93)
```

Comment:
Suggestion for clarity: is this the _fraction_ or the _percentage_? I'd suggest either changing the name to `percent` or changing the values to e.g. `0.7811` with a slight preference for the latter.